adds release scripts from gate.vaadin.com

paulroemer opened the pull request:
<!-- Reviewable:start -->
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/vaadin/framework/11928)
<!-- Reviewable:end -->

paulroemer committed:
adds release scripts from gate.vaadin.com

CLAassistant commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/not_signed)](https://cla-assistant.io/vaadin/framework?pullRequest=11928) <br/>Thank you for your submission! We really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla-assistant.io/vaadin/framework?pullRequest=11928) before we can accept your contribution.<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla-assistant.io/check/vaadin/framework?pullRequest=11928) it.</sub>

ZheSun88 committed:
Update isNewerVersion() method

apparently, we split major.minor.maintenance version to three parameters, but they are string instead of integer
for string comparison, 9 > 10, because 10 will be treated as 1 and 0; while for integer, 9<10. 
we have considered the case when maintenance version goes above 10, but not minor..

TatuLund committed:
Merge branch 'master' into adds-gate-release-files

TatuLund reviewed (approved):
<img class="emoji" title=":lgtm:" alt=":lgtm:" align="absmiddle" src="https://reviewable.io/lgtm.png" height="20" width="61"/>

> Reviewed 10 of 10 files at r1, 1 of 1 files at r2.
> *[Reviewable](https://reviewable.io/reviews/vaadin/framework/11928#-:-M3L6YmQB1G5cdQhGrrm:bnfp4nl)* status: :shipit: complete! all files reviewed, all discussions resolved



<!-- Sent from Reviewable.io -->

ZheSun88 reviewed (changes requested):
> *[Reviewable](https://reviewable.io/reviews/vaadin/framework/11928)* status: all files reviewed, 1 unresolved discussion (waiting on @paulroemer)

*[a discussion](https://reviewable.io/reviews/vaadin/framework/11928#-M3L7Woo26c4GfmL8DX7:-M3L7Wop93hx3jn_r6-F:brbz1gb) (no related file):*
currently, there is no need to merge this PR. 

all scripts used are in gate.vaadin.com .  

---


<!-- Sent from Reviewable.io -->

ZheSun88 commented:
closing this PR now
